cmd/tikv-ctl/src/cmd.rs

```diff
@@ -600,6 +600,33 @@ pub enum Cmd {
         #[structopt(long, default_value = "symlink")]
         rocksdb_files: String,
     },
+    /// flashback data in a TiKV to a certain version
+    ///
+    /// NOTE: Should use `./pd-ctl config set halt-scheduling true` to halt PD
```

Comment:
I have no problem with writing this in the comments, but I must remember to update the document to explain this to the user.

Reply:
I will update pd-ctl also.